### DISCUSSION TITLE
Storybook link should be near top of main README.

### DISCUSSION THREAD

**Discussion by rob-myers** [isaac-mason/recast-navigation-js#315] [General]
There's a lack of recast/detour examples available on the web.

So, I think your storybook link https://recast-navigation-js.isaacmason.com/ should be more prominent.

**Comment by isaac-mason** [comment]
That's fair, they are very hidden at the moment 😅 

I've added an "Examples" link to the top of the README: https://github.com/isaac-mason/recast-navigation-js/commit/d134336770f8983dc3cb9642c1f1251d6293cfe2
